index/sims4.toml

```diff
@@ -0,0 +1,8 @@
+name = "The Sims 4"
+home = "https://github.com/meaomeaomeaoo/Archipelago"
```

Comment:
And please replace this with the discord thread `https://discord.com/channels/731205301247803413/1079002955262480424`